service/featuregate/flags_test.go

```diff
@@ -96,33 +109,3 @@ func TestFlagValue_withPlus(t *testing.T) {
 		})
 	}
 }
-
-func TestFlagValue_SetSlice(t *testing.T) {
-	for _, tc := range []struct {
-		name     string
-		input    []string
-		expected FlagValue
-	}{
-		{
-			name:     "single item",
-			input:    []string{"foo"},
-			expected: FlagValue{"foo": true},
-		},
-		{
-			name:     "multiple items",
-			input:    []string{"foo", "-bar", "+baz"},
-			expected: FlagValue{"foo": true, "bar": false, "baz": true},
-		},
-		{
-			name:     "repeated items",
-			input:    []string{"foo", "-bar", "-foo"},
-			expected: FlagValue{"foo": true, "bar": false},
-		},
-	} {
-		t.Run(tc.name, func(t *testing.T) {
-			v := FlagValue{}
-			assert.NoError(t, v.SetSlice(tc.input))
```

Comment:
Could you please keep this, removing only when the public function is removed?

Reply:
Why? We test that via the "comma separated" cases, so the only untested code is the call from `SetSlice` to `setSlice`.

Reply:
Which is exactly what is being deprecated :-)

Reply:
I don't think I understand:

```golang
// fully tested
func Set() {
  // calls setSlice()
}

func SetSlice() {
  // Not tested 1 line
  return setSlice()
}

// fully tested
func setSlice() {}
}
```

Reply:
The "setSlice" is very well tested, it is only that deprecated func that is not tested which does very little (to nothing).